cli/src/cmd/forge/script/mod.rs

```diff
@@ -549,6 +549,12 @@ impl VerifyBundle {
             config_path: if config_path.exists() { Some(config_path) } else { None },
         };
 
+        // Default from `RetryArgs` struct.
+        if retry.delay.is_none() && retry.retries == 1 {
```

Comment:
I think we can just add these defaults directly on `RetryArgs` if they are more sane defaults for the create-related commands

Reply:
+1 on that, let's just bump the `Retry` defaults instead